queue/src/fiskaltrust.Middleware.Contracts/Repositories/FR/TempSpace/ftJournalFRCopyPayload.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace fiskaltrust.Middleware.Contracts.Repositories.FR.TempSpace
```

Comment:
Move this to `fiskaltrust.Middleware.Contracts.Models.FR` folder and namespace 🙏